src/qt/sendcoinsdialog.cpp

```diff
@@ -324,6 +325,13 @@ void SendCoinsDialog::on_sendButton_clicked()
     questionString.append(QString("<span style='font-size:10pt;font-weight:normal;'><br />(=%2)</span>")
         .arg(alternativeUnits.join(" " + tr("or") + "<br />")));
 
+    if (ui->optInRBF->isChecked())
+    {
+        questionString.append("<hr /><span style='color:#aa0000;'>");
+        questionString.append(tr("This transaction is replacable (optin-RBF)!"));
```

Comment:
This red warning with exclamation point is overly alarming.  It almost implies it's dangerous to you that its replaceable.  Transactions you send that are replaceable are not something to be warned about.  Worst case someone doesn't accept them, and you can replace it.